src/reanimated2/component/ScrollView.tsx

```diff
@@ -26,6 +26,13 @@ const AnimatedScrollView: AnimatedScrollViewFC = forwardRef(
         scrollViewOffset
       );
     }
+
+    if (!restProps.scrollEventThrottle) {
+      // Set default scrollEventThrottle to 16, because user expects
+      // to have continuous scroll events
+      restProps.scrollEventThrottle = 16;
```

Comment:
What do you think about setting 8 instead of 16 here? I don't think it would make a difference for 60 fps screens but it would work out-of-the-box for 120 fps devices.
```suggestion
      restProps.scrollEventThrottle = 8;
```